docs/getting-started.md

```diff
@@ -227,18 +266,18 @@ Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this
 
 #### Using Go
 
-Pilosa requires Go 1.12 or higher.
+Pilosa supports the two most recent versions of Go.
```

Comment:
"Pilosa follows the Go policy of supporting the two most recent major versions of Go."

FYI, as explained here: https://golang.org/doc/devel/release.html. The "major" is meaningful here.

Reply:
Ohhhhhhh